Content.Server/Objectives/Commands.cs

```diff
@@ -22,66 +22,70 @@ public void Execute(IConsoleShell shell, IPlayerSession? player, string[] args)
             {
-                var mind = data.ContentData()?.Mind;
-                if (mind == null)
-                {
-                    shell.SendText(player, "Can't find the mind.");
-                    return;
-                }
+                shell.SendText(player, "Can't find the playerdata.");
+                return;
+            }
 
-                if (!IoCManager.Resolve<IPrototypeManager>()
-                    .TryIndex<ObjectivePrototype>(args[1], out var objectivePrototype))
-                {
-                    shell.SendText(player, $"Can't find matching ObjectivePrototype {objectivePrototype}");
-                    return;
-                }
 
-                if (!mind.TryAddObjective(objectivePrototype))
-                {
-                     shell.SendText(player, "Objective requirements dont allow that objective to be added.");
-                }
+            var mind = data.ContentData()?.Mind;
+            if (mind == null)
+            {
+                shell.SendText(player, "Can't find the mind.");
+                return;
             }
-            else
+
+            if (!IoCManager.Resolve<IPrototypeManager>()
+                .TryIndex<ObjectivePrototype>(args[1], out var objectivePrototype))
             {
-                shell.SendText(player, "Can't find the playerdata.");
+                shell.SendText(player, $"Can't find matching ObjectivePrototype {objectivePrototype}");
+                return;
             }
+
+            if (!mind.TryAddObjective(objectivePrototype))
+            {
+                shell.SendText(player, "Objective requirements dont allow that objective to be added.");
+            }
+
         }
     }
 
     public class ListObjectivesCommand : IClientCommand
     {
         public string Command => "lsobjectives";
-        public string Description => "Lists all objectives in a players mind";
-        public string Help => "lsobjectives <username>";
+        public string Description => "Lists all objectives in a players mind.";
+        public string Help => "lsobjectives [<username>]";
         public void Execute(IConsoleShell shell, IPlayerSession? player, string[] args)
         {
-            var mgr = IoCManager.Resolve<IPlayerManager>();
-            if (mgr.TryGetPlayerDataByUsername(args[0], out var data))
+            IPlayerData? data;
+            if (args.Length == 0 && player != null)
             {
-                var mind = data.ContentData()?.Mind;
-                if (mind == null)
-                {
-                    shell.SendText(player, "Can't find the mind.");
-                    return;
-                }
+                data = player.Data;
+            }else if (!IoCManager.Resolve<IPlayerManager>().TryGetPlayerDataByUsername(args[0], out data))
+            {
+                shell.SendText(player, "Can't find the playerdata.");
+                return;
+            }
```

Comment:
This will throw if you call this from the server with no args (they have no player).
Also same line `}else if ()` :Stab: